### DISCUSSION TITLE
Add all modules to external

### DISCUSSION THREAD

**Discussion by n-e-o-n-s** [nklayman/vue-cli-plugin-electron-builder#1688] [Q&A] [answered]
Is there a way to have the electron-builder to copy all of my modules instead of having to add them in manually to external? I am currently forking a .js file named mls.js inside that file im requiring certain modules. So i am assuming because my main process does not reference these modules they are not being copied/bundled into the app.asar.  
` module.exports = {
 lintOnSave: false,
  pages: {
    index: "src/main.js",
    auth: "src/auth/main.js",
  },
  pluginOptions: {
    electronBuilder: {
      preload: "./src/preload.js",
      externals: [
        "electron-log",
        "electron-store,
        'axios',
      ],
      nodeModulesPath: ['./node_modules'],
      builderOptions: {
        extraResources: [
          {
            from: "./src/extraResources",
            to: "extraResources",
            filter: ["**/*"]
          }
        ],
      }
    },
  },
};` I have to keep adding what ever modules i add to my forked file.

**Comment by nklayman** [comment, marked as answer]
There's no way to add all dependencies as externals, as you'd always want some non-externals (ie vue). You just have to list them all in externals. However, V3 alpha of this plugin includes a much better native dependency detector which should make it unnecessary to list your native deps in externals.
